polaris-migrator/src/migrations/replace-sass-color/replace-sass-color.ts

```diff
@@ -0,0 +1,135 @@
+import type {FileInfo, API, Options} from 'jscodeshift';
+import postcss, {Plugin} from 'postcss';
+import valueParser from 'postcss-value-parser';
+import {colors as tokenColors} from '@shopify/polaris-tokens';
+
+import {
+  NamespaceOptions,
+  namespace,
+  isSassFunction,
+  getFunctionArgs,
+  stripQuotes,
+  StopWalkingFunctionNodes,
+} from '../../utilities/sass';
+import {isKeyOf} from '../../utilities/type-guards';
+
+import {
+  backgroundColorMap,
+  borderColorMap,
+  colorMap,
+  fillColorMap,
+} from './color-maps';
+
+const tokenColorsKeys = Object.keys(tokenColors);
+const maps = {
+  colorMap,
+  backgroundColorMap,
+  borderColorMap,
+  fillColorMap,
+};
+const propertyMap: {[key: string]: keyof typeof maps} = {
+  color: 'colorMap',
+  background: 'backgroundColorMap',
+  'background-color': 'backgroundColorMap',
+  border: 'borderColorMap',
+  'border-color': 'borderColorMap',
+  fill: 'fillColorMap',
+};
```

Comment:
```suggestion
const propertyMaps = {
  color: colorMap,
  background: backgroundColorMap,
  'background-color': backgroundColorMap,
  border: borderColorMap,
  'border-color': borderColorMap,
  fill: fillColorMap,
};
```

**question:** Is it possible to add the maps here and remove the intermediary `propertyMapKey` access?